website/docs/about/contribution.md

```diff
@@ -1,34 +1,200 @@
+
+!!!
+
+### Request
+
+The request of collaboration can come:
+
+- from a consumer team or a single product designer/developer/manager, because they think there is an opportunity of contributing to the design system or [collaborating]((#forms-of-collaboration)) with the design system team.
+
+!!! Insight
+
+We offer [multiple contact points](/about/support) that can be used to initiate the conversation.
+
 !!!
 
-## Types of contribution
+- from the HDS team, because we may see an opportunity to collaborate with a product team on a specific project to bring some of the work back into the design system
+
+### Evaluation
+
+The HDS team will evaluate the request from the consumers’ team (or the opportunity of re-using part of the consumers’ work) in terms of:
+
+- Reusability across different, multiple products (the most important factor for us)
+- Relevance and added value for the design system as a whole
+- Integration with existing foundations/components/features of the system
+- Impact on existing components (especially in case of breaking changes)
+- Expected effort and possible complexities and unknowns involved
+- Availability of resources in terms of HDS engineers and designers
+- Prioritization in relation with other initiatives already undertaken and/or planned.
+
+### Decision
+
+There are different possible outcomes of the initial evaluation:
+- The HDS team decides to start (or plans) to **dedicate time and resources to the work/collaboration**. In this case a kickoff meeting is organized to define the details of such collaboration (see below).
+- The HDS team decides that there’s an **opportunity for collaboration in the future**, that it would add value to the system for all the consumers but in that specific moment doesn’t have time/resources to dedicate to it. In this case the HDS team adds a new task/epic to their backlog to re-evaluate this activity in the (near) future.
+- The HDS team determines that they **can't support this request/work** (this can be for a variety of reasons, it's not possible to enumerate them all here). In this case the HDS team will provide guidance and suggestions on how to move forward, but the final decision on how to proceed with their own work will be left to the consumer team.
+
+### Kickoff
+
+In case the HDS team has decided to dedicate time and resources to this project it will organize a meeting with the consumers’ team and share the decision about the suggested/proposed collaboration, to define together:
+
+- What resources will be allocated to the project (on both sides, HDS and consumer team) in terms of designers and engineers
+- What will be the scope of the project (or if an initial discovery phase is necessary to better define this scope) and if the project can be delivered incrementally
+- What is the timeline for the project, if the consumer team has a deadline to meet, if the effort needs to be timeboxed (this will be strictly related with the definition of its scope)
```

Comment:
```suggestion
- What is the timeline for the project, if the consumer team has a deadline to meet, and if the effort needs to be timeboxed (this will be strictly related to the definition of its scope)
```